include/mega/filesystem.h

```diff
@@ -339,7 +339,7 @@ struct MEGA_API FileSystemAccess : public EventTrigger
     bool target_exists;
 
     // append local operating system version information to string
-    virtual void osversion(string*) const { }
+    virtual void osversion(string*, bool includeArchitecture) const { }
```

Comment:
the name is deceiving I'm afraid: in the end it only includes info when using the Wow64 emulator ...
May I suggest: `includeArchExtraInfo` (`includeArchExtraInfoWhenSignificant` is too large ;-) ), and adding a word on WinFileSystemAccess::osversion prototype regarding `WOW64` ...